docs/source/starter/new-project.rst

```diff
@@ -61,8 +102,33 @@ You could also use conda environments
     conda activate my_env
     pip install pytorch-lightning
 
+Install future patch release from the source
```

Comment:
this is the quick start guide, meaning the user will install it fresh, so they anyway get the 1.5.x version. Including this upgrade process here is redundant.

Reply:
what we can do is double branches, meaning the actual `release/1.5.x` will be called `release/stable` and when we start with v1.6, the rename `release/stable` back to `release/1.5.x`